packages/mask/src/plugins/Predict/Azuro/helpers/param.ts

```diff
@@ -0,0 +1,13 @@
+const param: Record<number, number> = {
+    1: 0,
```

Comment:
Replace keys to use enums.

Reply:
like this ?
`
export enum {
One = 1
Four = 4
...
}
`

Reply:
And please give them a meaningful name. what are those numbers used for?

Reply:
They are ids for params. It's a standard in the industry. e.g. `4: 1.5` for over 1.5 goals in a game.

Reply:
I don't get it. Can you give me a simple example using `enum` structure?

Reply:
Done.